ext/drcontainers/hashtable.c

```diff
@@ -457,6 +457,22 @@ hashtable_remove_range(hashtable_t *table, void *start, void *end)
     return res;
 }
 
+
+void
+hashtable_apply_to_all_payloads(hashtable_t *table, void (*apply_func)(void *payload))
+{
+    DR_ASSERT(apply_func);
```

Comment:
style: explicit bools: != NULL

Reply:
Could you also use DR_ASSERT_MSG and provide a message when this assertion is hit?